website/pages/docs/plugins/sources/overview.mdx

```diff
@@ -30,6 +30,7 @@ Official source plugins follow [release stages](#source-plugin-release-stages).
             { name: "Datadog", stage:"Preview" },
             { name: "Tailscale", stage:"Preview" },
             { name: "Vercel", stage:"Preview" },
+            { name: "Snyk", stage:"Preview" },
```

Comment:
Add it after Datadog so preview plugins are sorted as a subset?

Reply:
then whole preview part still will require sorting (cloudflare w=needs to go on top for that part?)